packages/docutils/lib/cli/check.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * Provides functions to validate user-provided options as part of Yargs' post-parsing "check" phase
+ * @module
+ */
+
+import {fs} from '@appium/support';
+import _ from 'lodash';
+import pluralize from 'pluralize';
+import {InferredOptionTypes, Options} from 'yargs';
+import logger from '../logger';
+
+const log = logger.withTag('check');
+
+/**
+ * Given a list of objects with `id` and `path` props, filters out the ones that do not exist
+ * @param paths Filepaths
+ * @returns Missing files
+ */
+async function filterMissing(paths: MissingFileData[]): Promise<MissingFileData[]> {
+  const exists = await Promise.all(
+    paths.map(async ({id, path}) => ({id, path, exists: await fs.exists(path)}))
+  );
+  const results = _.reject(exists, 'exists');
+  return _.map(results, (result) => _.omit(result, 'exists'));
```

Comment:
I assume bluebird's filter could do this all in one line

Reply:
I could do this on one line with lodash too, but it was too clever